geode-core/src/main/java/org/apache/geode/pdx/internal/PeerTypeRegistration.java

```diff
@@ -107,18 +110,21 @@ public PeerTypeRegistration(InternalCache cache) {
     if (distributedSystemId == -1) {
       distributedSystemId = 0;
     }
-    this.dsId = distributedSystemId << 24;
-    this.maxTypeId = 0xFFFFFF;
+    this.distributedSystemId = distributedSystemId << 24;
+    maxTypeId = 0xFFFFFF;
```

Comment:
it's better to be this.maxTypeId = 0xFFFFFF;

Reply:
`this` is redundant in that context.

Reply:
Did some readability cleanup on the class, so the members being set from parameters are more clear.